include/osquery/config/packs.h

```diff
@@ -0,0 +1,101 @@
+/*
+ *  Copyright (c) 2014, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#pragma once
+
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include <boost/property_tree/ptree.hpp>
+
+#include <osquery/database.h>
+#include <osquery/flags.h>
+#include <osquery/registry.h>
+#include <osquery/status.h>
+
+namespace osquery {
+
+typedef boost::property_tree::ptree ptree;
```

Comment:
We've been using [namespace aliases](https://github.com/facebook/osquery/blob/master/osquery/filesystem/darwin/plist.mm#L21), any reason for wanting the typedef instead?

Could write: `namespace pt = boost::property_tree` then you get a nice `pt::ptree` structure.